### ISSUE TITLE
will it work with Ventura?

### ISSUE THREAD

**Issue by tele-bird** [MicrosoftDocs/visualstudio-docs#8799]
This simple question has two seemingly different answers within the documentation here:

[Visual Studio 2022 for Mac System Requirements](https://learn.microsoft.com/en-us/visualstudio/releases/2022/mac-system-requirements)

Under the Supported Operating Systems section, there's a note that says:

`macOS Ventura is not yet supported in the latest stable version of Visual Studio 2022 for Mac`

However, the very next section contains this:

`Visual Studio 2022 for Mac is supported on the following 64-bit operating systems:`
`- macOS Ventura 13.0 or higher.`
`- macOS Monterey 12.0 or higher.`
`- macOS Big Sur 11.0 or higher.`

Sooo, which is it??  Supported or unsupported?

**Comment by WilliamAntonRohm** [comment]
@tele-bird -- Phil, thank you for your questions.  Please consider these resources:
- [Visual Studio on Microsoft Q&A](https://docs.microsoft.com/answers/products/vs) 
- [Visual Studio MSDN Forum](https://social.msdn.microsoft.com/Forums/vstudio) 
- [Stack Overflow](https://stackoverflow.com/search?q=%22visual+studio%22)  
- [Reddit](https://www.reddit.com/r/VisualStudio)

@jmatthiesen -- Jordan, please look into this issue.

**Comment by tele-bird** [comment]
I noticed this language in the [release notes for VS4Mac 17.4.4](https://learn.microsoft.com/en-us/visualstudio/releases/2022/mac-release-notes#1744--visual-studio-2022-for-mac-v1744-newreleasebutton):

_Slow editor performance on macOS Ventura has been significantly improved_

This implies that VS4Mac is supported, so I went ahead and installed this version and upgraded to Ventura, and it works!
Though I'm still disappointed about the lack of clarity in your documentation.  We devs have to be careful with upgrades in particular since minor bugs can be a complete blocker for development, and it is rather difficult (impossible?) to downgrade macOS.
